config.test.yaml

```diff
@@ -157,6 +157,10 @@ services:
       spec: *spec
       salt: secret
       default_page_size: 1
+      contentTypes:
```

Comment:
I'd prefer for this to be inside a `test` hash so that it is clear is pertains only to tests, something like:

```yaml
test:
  content_types:
    ...
```

(also note how we do not use camel case in the config)